webapp/channels/src/components/channel_invite_modal/channel_invite_modal.tsx

```diff
@@ -271,22 +271,20 @@ export default class ChannelInviteModal extends React.PureComponent<Props, State
                 />
                 <div className='more-modal__details'>
                     <div className='more-modal__name'>
-                        <span>
-                            {displayName}
+                        <span className='d-flex'>
```

Comment:
Styled components instead?

Reply:
Since a class already exists, I was thinking just applying the class may be better taking a utility classes approach rather than applying it as css?

Reply:
The problem with that approach is its really hard to figure out what is going on sometimes. Where as with styled components it's very clear and unambiguous.